### ISSUE TITLE
Haven't been able to "bibm search" for authors with spaces in their last names

### ISSUE THREAD

**Issue by AndresErnestoGuzman** [pcubillos/bibmanager#69]
For example
@ARTICLE{2008A&A...485..137C,
       author = {{Caratti o Garatti}, A. and {Froebrich}, D. and {Eisl{\"o}ffel}, J. and
         {Giannini}, T. and {Nisini}, B.},
        title = "{Molecular jets driven by high-mass protostars: a detailed study of the IRAS 20126+4104 jet}",
      journal = {\aap},
     keywords = {stars: pre-main-sequence, infrared: ISM, ISM: jets and outflows, ISM: kinematics and dynamics, individual objects: IRAS 20126+4104, Astrophysics},
         year = 2008,
        month = jul,
       volume = {485},
       number = {1},
        pages = {137-152},
          doi = {10.1051/0004-6361:200809515},
archivePrefix = {arXiv},
       eprint = {0803.2012},
 primaryClass = {astro-ph},
       adsurl = {https://ui.adsabs.harvard.edu/abs/2008A&A...485..137C},
      adsnote = {Provided by the SAO/NASA Astrophysics Data System}
}

I get
`bibm search`
`(Press 'tab' for autocomplete)`
`author:"^Caratti o Garatti"`
``

**Comment by pcubillos** [comment]
Hey!,
The problem is that the search command is following the BibTeX format rules for author names, which are not always unambiguous to parse what is First/Last names.  The short answer is:  you can use curly brackets to make clear what is the 'last name' part of the entry:

```shell
bibm search
(Press 'tab' for autocomplete)
author:"^{Caratti o Garatti}"

Title: Molecular jets driven by high-mass protostars: a detailed study of the
       IRAS 20126+4104 jet, 2008
Authors: {Caratti o Garatti}, A.; et al.
key: 2008A&A...485..137C
```

The long answer is a bit in the [search documentation](https://bibmanager.readthedocs.io/en/latest/bibtex.html#search), but not explicitly.  There, there's a link to the BibTeX author formatting (http://mirror.easyname.at/ctan/info/bibtex/tamethebeast/ttb_en.pdf).  This format assumes that the names are given in a 'First von Last' format.  In your entry, the code is guessing that the lowercase 'o' is the 'von' section of the name (First: Caratti, von:o, Last:Garatti).  So the solution is to use the curly brackets to specify that the whole text is the last name.

I'll see if I can make the documentation more clear on this without getting too wordy.
Thanks!
